src/dev/build/tasks/create_archives_task.js

```diff
@@ -71,5 +90,20 @@ export const CreateArchivesTask = {
           throw new Error(`Unexpected extension for archive destination: ${destination}`);
       }
     }
+
+    const reporter = CiStatsReporter.fromEnv(log);
+    if (reporter.isEnabled()) {
+      await reporter.metrics(
+        await Promise.all(
+          archives.map(async ({ format, path }) => {
+            return {
+              group: `${build.isOss() ? 'oss ' : ''}distributable size`,
+              id: format,
+              value: (await asyncStat(path)).size,
```

Comment:
If what format & units we report `size`? I hope we won't have to convert GB <--> MB in the storage 😅